sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -164,4 +166,20 @@ object SQLExecution {
       }
     }
   }
+
+  /**
+   * Wrap passed function to ensure necessary thread-local variables like
+   * SparkContext local properties are forwarded to execution thread
+   */
+  def withThreadLocalCaptured[T](sparkSession: SparkSession, exec: ExecutionContext)(
+    body: => T): Future[T] = {
```

Comment:
the indentation looks weird. How about
```
def withThreadLocalCaptured[T](
    sparkSession: SparkSession, exec: ExecutionContext)(body: => T): Future[T]
```

Reply:
Ok, weird style was because of `scalafmt`, now as per suggestion i have manually formatted and updated the PR